Manager/src/test/java/io/raspberrywallet/manager/database/DatabaseTest.java

```diff
@@ -85,18 +80,17 @@ void deserialize() {
         }
 
         /*
-        * `List::containsAll` uses `Object::equals` to determine whether a collection contains another.
-        * This condition checks if `List`s are equal using overridden `Object::equal`.
-        * */
+         * `List::containsAll` uses `Object::equals` to determine whether a collection contains another.
+         * This condition checks if `List`s are equal using overridden `Object::equal`.
+         * */
         assertEquals(newWallet, walletEntity1);
     }
 
     @Test
     void encrypt() {
-
         try {
             encrypted = db.encrypt(db.getSerialized());
```

Comment:
Shouldn't we use JUnit 5 Exceptions handling?
https://howtodoinjava.com/junit5/expected-exception-example/

Version with lamda expression would be more clear.

Reply:
Sorry, but that's not my code here is PR for that #12